kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/junit/MethodTestDescriptor.kt

```diff
@@ -6,7 +6,8 @@ import org.junit.platform.engine.support.descriptor.AbstractTestDescriptor
 import org.junit.platform.engine.support.descriptor.MethodSource
 
 internal class MethodTestDescriptor(
-    val testMethod: Method, parent: TestDescriptor,
+    val testMethod: Method,
+    parent: TestDescriptor,
```

Comment:
Consider going to the original formatting so that we will not have any unnecessary conflicts in the future (e.g. with the PR about separating regression)